diff --git a/dist/lib/dprint/dprint.js b/dist/lib/dprint/dprint.js
index 124201205c4c8f12942bae72018e4bc6351d5c2f..d73dc4f147957331d4bdd29b9451f7947d4312a5 100644
--- a/dist/lib/dprint/dprint.js
+++ b/dist/lib/dprint/dprint.js
@@ -117,7 +117,8 @@ function getFormatter(filePath, configName, configFile, log = true) {
             return formatter;
         }
         else if (isPluginName(configName) && log) {
-            console.warn("File %s not supported by %s", filePath, pluginsName[configName]);
+            // silenced to reduce noise
+            //console.warn("File %s not supported by %s", filePath, pluginsName[configName]);
         }
     }
     else if (isPluginName(configName) && log) {
