diff --git a/dist/index.cjs b/dist/index.cjs
index 3439ef5ffee37f73f39e3451d1971af67f9b9cc2..6026d1cdc40948e29a55c8e14850033488f331ab 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -2997,6 +2997,8 @@ function vuePlugin(rawOptions = {}) {
       const ssr = opt?.ssr === true;
       const { filename, query } = parseVueRequest(id);
       if (query.vue) {
+        // patroza: workaround for proxying a vite project under a nuxt path
+        if (filename.startsWith("/__skip_vite/configurator-app/")) return 
         if (query.src) {
           return fs__default.readFileSync(filename, "utf-8");
         }
diff --git a/dist/index.mjs b/dist/index.mjs
index 6c18c291efbda51454ee5a3d30ed0368e7a4688e..5e075362c5232f859f6bfe783314f27926324ed5 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2984,6 +2984,8 @@ function vuePlugin(rawOptions = {}) {
       const ssr = opt?.ssr === true;
       const { filename, query } = parseVueRequest(id);
       if (query.vue) {
+        // patroza: workaround for proxying a vite project under a nuxt path
+        if (filename.startsWith("/__skip_vite/configurator-app/")) return 
         if (query.src) {
           return fs.readFileSync(filename, "utf-8");
         }
