diff --git a/dist/index.cjs b/dist/index.cjs
index 68cd520611ca5ea7bf29a2bb789b21615fa45714..a5a1702ffd49a13f4cbcde56f686734abd64e1ed 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -2967,6 +2967,8 @@ function vuePlugin(rawOptions = {}) {
       }
       const { filename, query } = parseVueRequest(id);
       if (query.vue) {
+        // patroza: workaround for proxying a vite project under a nuxt path
+        if (filename.startsWith("/__skip_vite/configurator-app/")) return 
         if (query.src) {
           return fs__default.readFileSync(filename, "utf-8");
         }
diff --git a/dist/index.mjs b/dist/index.mjs
index 739335e4bad7f9356669f2d585cb44435fc615d6..9edb1126ac3ca98f199a37f153e9462b58f2ef1e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2954,6 +2954,8 @@ function vuePlugin(rawOptions = {}) {
       }
       const { filename, query } = parseVueRequest(id);
       if (query.vue) {
+        // patroza: workaround for proxying a vite project under a nuxt path
+        if (filename.startsWith("/__skip_vite/configurator-app/")) return 
         if (query.src) {
           return fs.readFileSync(filename, "utf-8");
         }
