diff --git a/dist/index.js b/dist/index.js
index c15c7abc522e45d65c43dff265a22082a5b3dbc0..7e3172a101dde25f31d8e3f2a2eb6e2d7fa9ebd7 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2023,6 +2023,8 @@ function vuePlugin(rawOptions = {}) {
 				const ssr = opt?.ssr === true;
 				const { filename, query } = parseVueRequest(id);
 				if (query.vue) {
+          // patroza: workaround for proxying a vite project under a nuxt path
+          if (filename.startsWith("/__skip_vite/configurator-app/")) return 
 					if (query.src) return fs.readFileSync(filename, "utf-8");
 					const descriptor = getDescriptor(filename, options.value);
 					let block;
