diff --git a/dist/index.mjs b/dist/index.mjs
index 09d3418894dfc0cb747c5f68cb3c74554a2c1e79..71940fb0269502ccc9f95b4ee83b8202ae7e8479 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1700,6 +1700,8 @@ function vuePlugin(rawOptions = {}) {
 				const ssr = opt?.ssr === true;
 				const { filename, query } = parseVueRequest(id);
 				if (query.vue) {
+          // patroza: workaround for proxying a vite project under a nuxt path
+          if (filename.startsWith("/__skip_vite/configurator-app/")) return 
 					if (query.src) return fs.readFileSync(filename, "utf-8");
 					const descriptor = getDescriptor(filename, options.value);
 					let block;
