diff --git a/dist/vuetify-labs.cjs b/dist/vuetify-labs.cjs
index d021d0ea0f93f5f2f3817781c588dba0e67dee6b..66bdedf700ceb94b8da674a65f44af9cda8c3562 100644
--- a/dist/vuetify-labs.cjs
+++ b/dist/vuetify-labs.cjs
@@ -15287,7 +15287,7 @@
       vue.watch(isActive, async val => {
         if (!val) {
           await vue.nextTick();
-          overlay.value.activatorEl?.focus({
+          overlay.value?.activatorEl?.focus({
             preventScroll: true
           });
         }
diff --git a/dist/vuetify-labs.esm.js b/dist/vuetify-labs.esm.js
index 8fca999ff4fcef2a9ae11b77d396dcb396a5241f..97c0e2ea994d31c51bc9e7ad388dcc6b7a80a779 100644
--- a/dist/vuetify-labs.esm.js
+++ b/dist/vuetify-labs.esm.js
@@ -15283,7 +15283,7 @@ const VDialog = genericComponent()({
     watch(isActive, async val => {
       if (!val) {
         await nextTick();
-        overlay.value.activatorEl?.focus({
+        overlay.value?.activatorEl?.focus({
           preventScroll: true
         });
       }
diff --git a/dist/vuetify-labs.js b/dist/vuetify-labs.js
index d021d0ea0f93f5f2f3817781c588dba0e67dee6b..66bdedf700ceb94b8da674a65f44af9cda8c3562 100644
--- a/dist/vuetify-labs.js
+++ b/dist/vuetify-labs.js
@@ -15287,7 +15287,7 @@
       vue.watch(isActive, async val => {
         if (!val) {
           await vue.nextTick();
-          overlay.value.activatorEl?.focus({
+          overlay.value?.activatorEl?.focus({
             preventScroll: true
           });
         }
diff --git a/dist/vuetify.cjs b/dist/vuetify.cjs
index 566ff2408308ec205d95f94b3b3eb951dd86b768..82fdbf035657a597b257a5fef643deb2cf8c8ce1 100644
--- a/dist/vuetify.cjs
+++ b/dist/vuetify.cjs
@@ -15616,7 +15616,7 @@
       vue.watch(isActive, async val => {
         if (!val) {
           await vue.nextTick();
-          overlay.value.activatorEl?.focus({
+          overlay.value?.activatorEl?.focus({
             preventScroll: true
           });
         }
diff --git a/dist/vuetify.esm.js b/dist/vuetify.esm.js
index 660e31b170c94f07d52509dc1971ce2acec6017f..650ca6cbf80dcc202f4108a44963fb74a749b148 100644
--- a/dist/vuetify.esm.js
+++ b/dist/vuetify.esm.js
@@ -15612,7 +15612,7 @@ const VDialog = genericComponent()({
     watch(isActive, async val => {
       if (!val) {
         await nextTick();
-        overlay.value.activatorEl?.focus({
+        overlay.value?.activatorEl?.focus({
           preventScroll: true
         });
       }
diff --git a/dist/vuetify.js b/dist/vuetify.js
index c0b8cab875072a7eb22e0a25cbd3794d152750d5..a3e02d88818cd87673a9401f70402ece2e350a00 100644
--- a/dist/vuetify.js
+++ b/dist/vuetify.js
@@ -15616,7 +15616,7 @@
       vue.watch(isActive, async val => {
         if (!val) {
           await vue.nextTick();
-          overlay.value.activatorEl?.focus({
+          overlay.value?.activatorEl?.focus({
             preventScroll: true
           });
         }
diff --git a/lib/components/VDialog/VDialog.js b/lib/components/VDialog/VDialog.js
index cbe6ac2f6fb733d0b010041894306e6c59a5c751..31eb21b9ef53e8c3cea9f03f433353273a1de480 100644
--- a/lib/components/VDialog/VDialog.js
+++ b/lib/components/VDialog/VDialog.js
@@ -58,7 +58,7 @@ export const VDialog = genericComponent()({
     watch(isActive, async val => {
       if (!val) {
         await nextTick();
-        overlay.value.activatorEl?.focus({
+        overlay.value?.activatorEl?.focus({
           preventScroll: true
         });
       }
